modules/platforms/dotnet/Apache.Ignite.Core.Tests/Cache/CacheLocalActiveTransactionsTest.cs

```diff
@@ -0,0 +1,234 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Apache.Ignite.Core.Tests.Cache
+{
+    using System;
+    using System.Linq;
+    using Apache.Ignite.Core.Cache;
+    using Apache.Ignite.Core.Cache.Configuration;
+    using Apache.Ignite.Core.Impl.Transactions;
+    using Apache.Ignite.Core.Transactions;
+    using NUnit.Framework;
+    using NUnit.Framework.Constraints;
+
+    /// <summary>
+    /// Tests <see cref="ITransactions.GetLocalActiveTransactions"/>.
+    /// </summary>
+    public class CacheLocalActiveTransactionsTest : TestBase
+    {
+        private const string IgniteInstanceName = "GetLocalActiveTransactionsTest";
+
+        /// <summary>
+        /// Fixture setup.
+        /// </summary>
+        [TestFixtureSetUp]
+        public void SetUp()
+        {
+            IgniteConfiguration cfg = new IgniteConfiguration(GetConfig())
+            {
+                IgniteInstanceName = IgniteInstanceName
+            };
+
+            Ignition.Start(cfg);
+        }
+
+        /// <summary>
+        /// Fixture teardown.
+        /// </summary>
+        [TestFixtureTearDown]
+        public void TearDown()
+        {
+            Ignition.StopAll(true);
+        }
```

Comment:
I think we can remove this part and use `Ignition.GetIgnite()` without arguments below since only one Ignite instance is being used.

Reply:
Done.